agent-tooling/src/main/java/io/opentelemetry/auto/tooling/matcher/AdditionalLibraryIgnoresMatcher.java

```diff
@@ -116,7 +116,11 @@ public boolean matches(final T target) {
             || name.startsWith(
                 "org.springframework.boot.context.embedded.tomcat.TomcatEmbeddedServletContainer$")
             || name.equals(
-                "org.springframework.boot.context.embedded.tomcat.TomcatEmbeddedWebappClassLoader")) {
+                "org.springframework.boot.context.embedded.tomcat.TomcatEmbeddedWebappClassLoader")
```

Comment:
I would love to see some explanations, why these class loaders in particular are listed.

Reply:
this was not changed in this PR, but class loaders are listed in here so that `java-classloader` instrumentation will not be skipped. the way these are caught and added here is that `AgentTestRunner` does not use `AdditionalLibraryIgnoresMatcher`, and then it validates that there are no classes that were instrumented that would have been skipped by `AdditionalLibraryIgnoresMatcher`, and so the test fails, and then you add suppress the class from being skipped in here